code/__HELPERS/atmospherics.dm

```diff
@@ -0,0 +1,8 @@
+/proc/cmp_epsilon_less_than(a,b,epsilon = MAXIMUM_ERROR_GAS_REMOVAL)
+	return (a < (b + epsilon))
+
+/proc/cmp_epsilon_greater_than(a,b,epsilon = MAXIMUM_ERROR_GAS_REMOVAL)
+	return ((a + epsilon) > b)
+
+/proc/cmp_epsilon_equals(a,b,epsilon = MAXIMUM_ERROR_GAS_REMOVAL)
+	return (((a + epsilon) > b) && ((a - epsilon) < b))
```

Comment:
Call it like, molar compare or something, and remove the epsilon arg

Reply:
MAXIMUM_ERROR_GAS_REMOVAL is like 2.2e-3 which well while good for most purposes is way overkill for cases involving a single gas type (eg. perhaps filter / breath code) so I would personally consider keeping it, besides, if you don't specify an epsilon it'll default to that anyways, at the cost of well presumably slight performance differences

Reply:
alright that's good enough for me